packages/tpetra/core/src/Tpetra_Details_packCrsMatrix_def.hpp

```diff
@@ -796,12 +796,34 @@ packCrsMatrix (const CrsMatrix<ST, LO, GO, NT>& sourceMatrix,
   }
 
   if (num_export_lids == 0) {
```

Comment:
I believe all the fences in this file can safely be removed.

Reply:
Removed #7545